wp/lib/bap_wp/src/compare.ml

```diff
@@ -169,15 +169,17 @@ let compare_subs
 
 let compare_subs_empty : comparator * comparator =
   let postcond ~original:(_, env1) ~modified:(_,env2) ~rename_set:_ =
-    let post = Env.trivial_constr env1 in
+    let ctx = Env.get_context env1 in
+    let post = Constr.trivial ctx in
     post, env1, env2
   in
   let hyps = postcond in
   postcond, hyps
 
 let compare_subs_empty_post : comparator * comparator =
   let postcond ~original:(_, env1) ~modified:(_, env2) ~rename_set:_ =
-    let post = Env.trivial_constr env1 in
+    let ctx = Env.get_context env1 in
```

Comment:
?

Reply:
I moved the trivial constraint from `environment.ml` to `constraint.ml` because this was used in the loop unroller and since I moved the loop unroller to `precondition.ml`, keeping that function in `environment.ml` would cause cyclical dependencies. I also think trivial fits `constraint.ml` better than `environment.ml`.

Reply:
Ok, makes sense!